assayist/processor/utils.py

```diff
@@ -1,16 +1,19 @@
 # SPDX-License-Identifier: GPL-3.0+
 
+import json
 import os
-import subprocess
 import shutil
+import subprocess
+import tarfile
```

Comment:
Do we need to add anything to requirements.txt to cover gzip, xz compression cases? I can't remember if these are always installed on the system.